pkg/test/resource/resource.go

```diff
@@ -367,11 +380,18 @@ func (ts TestSnapshot) Generate() cache.Snapshot {
 		listeners[i] = listener
 	}
 
+	runtimes := make([]cache.Resource, ts.NumRuntimes)
+	for i := 0; i < ts.NumRuntimes; i++ {
+		name := fmt.Sprintf("runtime-%d", i)
```

Comment:
Not sure if this is an appropriate ask, but can RTDS resources be fully described by the API? Currently, the name of the RTDS runtime layers exists in two places: in the RTDS resource returned by the RTDS API here, and in the static  config file (see change below). This creates a coupling between the RTDS API and the config file. Hope I'm making sense :)